src/Nexus/PackageManagement/PackageController.cs

```diff
@@ -497,7 +497,7 @@ private async Task<string[]> DiscoverGithubReleasesAsync(CancellationToken cance
             continue;
         }
 
-        return [.. result];
+        return result.ToArray();
```

Comment:
I am not sure anymore but now that I see it, I find `result.ToArray();` much more readable because the task is to convert from `List<string>` to `string[]`. The spreading operator (`.. `) is very new and I think it is nice to have when combining multiple Lists / Arrays as shown here:

![grafik](https://github.com/user-attachments/assets/7d68f82d-0eba-4511-9ec9-4aa0da7f783f)

But to convert a list into an array I like the old school `.ToArray()` more.